### ISSUE TITLE
Pin voyager edge port

### ISSUE THREAD

**Issue by kaitmore** [DecipherNow/helm-charts#382]

**Comment by danielpcox** [comment]
This would simplify tutorials and training material, and it's a trivial fix.

Kait posted how to do it, by adding `nodePort`s to the right file.

```
# Source: greymatter/charts/edge/templates/edge-ingress.yaml

apiVersion: voyager.appscode.com/v1beta1
kind: Ingress
metadata:
  name: edge
  namespace: default
  annotations:
    kubernetes.io/ingress.class: "voyager"
    ingress.appscode.com/type: NodePort
    ingress.appscode.com/ssl-passthrough: "true"
spec:
  rules:
    - tcp:
        port: "80"
        nodePort: "30001"
        backend:
          serviceName: edge
          servicePort: 8080
    - tcp:
        port: "443"
        nodePort: "30000"
        backend:
          serviceName: edge
          servicePort: 8080
```

**Comment by kaitmore** [comment]
Closed by #385
